src/components/proximities/proximities.tsx

```diff
@@ -0,0 +1,16 @@
+import { fetchProperties } from '../../api/fetch-properties';
+import { LoadingError } from '../loading/loading-error';
+import { ProximitiesSection } from './proximities-section';
+
+export const Proximities = () => {
+  const { data = [], loading, error } = fetchProperties();
+  const proximities = data?.[0]?.proximities || [];
+  const latitude = data?.[0]?.address.latitude || 0;
+  const longitude = data?.[0]?.address.longitude || 0;
```

Comment:
a diferenca entre o OU `||` o nullish coalecing `??` é que o `??` testa se a primeira parte da verificacao é `null` ou `undefined` (daí o nullish) para utilizar a segunda. O `||` testa se a primeira parte é `true` ou `false`. Nos casos de string vazia (`""`), 0 etc, o `||` retornaria a segunda parte, pq eles apresentam o valor `false`.

Nesses casos, faria sentido colocar essas comparações apenas para valores nullish.

```suggestion
  const proximities = data?.[0]?.proximities ?? [];
  const latitude = data?.[0]?.address.latitude ?? 0;
  const longitude = data?.[0]?.address.longitude ?? 0;
```